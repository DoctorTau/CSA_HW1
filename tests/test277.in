681
9967 5981 -1788 4292 -4809 6246 -1603 -2543 6390 -285 -9729 -4110 3592 -8755 1869 -8137 511 3277 4172 5104 6575 -4085 1183 -7399 1377 -3740 -5963 -6430 -1151 -2846 -2086 7962 -4525 -9927 -9204 1139 -2486 5561 9455 3522 2189 1534 -9722 -9663 1602 -7311 6426 -2345 4053 -7829 729 -3106 3485 5358 -4382 -8657 -651 7733 -8134 8555 1245 1629 -3996 8137 3371 -7771 -8895 2111 -9737 -7335 -3745 9586 -1123 -6491 -1673 9280 -9229 2492 -3346 8028 -503 -9710 -5603 -5448 -3466 9267 8823 6807 -7533 8708 9113 -5380 -4363 -8944 4360 -7868 -1643 4985 -31 7046 -3207 5640 1350 4120 -1474 -9942 -5494 1350 2492 177 1817 -2938 5707 -9121 -1206 -7355 3739 -766 1587 8583 -9693 5962 3148 -3250 -4679 9569 -1616 2968 -142 4098 1054 7728 870 775 -9097 -7834 501 -1149 -2298 -2905 -996 -3601 -4454 6699 7265 6867 -6697 27 -4787 -2499 -7483 714 4014 2175 -5224 -5643 2683 -1135 978 5004 3086 -3118 8767 2355 8266 -3884 6261 7884 -2645 -569 -8218 8252 -1073 -5039 7626 2572 7705 -4074 5758 -3240 -2285 -2257 -8942 2398 5133 6029 7021 -192 -8865 -4854 -9411 3126 -3721 9568 -4033 7092 5199 -4690 942 9089 3187 1520 -5327 8014 -9395 -5040 4556 6806 3877 -134 -1946 -2662 -8268 -324 7314 -1001 -7466 -5825 596 -6414 -9137 3944 1660 -3920 8605 9043 -3656 1698 -8118 1507 2221 -8154 2635 -1981 4888 4887 9862 7326 -7056 -1233 -9590 5196 7311 -6550 -3298 4330 -1431 -444 367 -8255 -8848 -8224 -5845 3836 8026 751 4715 1636 -2507 1727 3334 4353 1567 2947 -4231 -9032 -4824 -9921 -184 -4457 -6865 7308 5927 -2295 4832 -8356 -5271 -6406 -7070 7692 -9614 -8724 7128 7678 -9012 -506 9910 -5106 -7046 4821 -3295 4268 -8054 -4461 -5207 -3409 4052 -2843 -7114 4265 -1129 3512 -806 4288 8409 5959 -6542 4795 -3233 -6767 -2957 -4837 -3546 1411 3226 8730 -638 6415 7177 5759 -4694 -9305 -788 7000 5653 7429 -2506 3100 -8261 7286 4549 5500 -564 5296 -1234 -8178 -2851 -7458 -1632 -360 -3138 -3817 -5112 1324 9015 -6774 39 -5211 -5188 5845 6328 3735 9935 180 -9415 -1760 -6870 7042 -7105 -9872 -5840 -4072 559 -9314 -7211 6761 3127 685 -4659 1985 -2545 -6981 -7399 -499 4275 7610 9261 9636 -2547 -6259 -6618 7964 2278 331 -1347 -7122 5081 -1917 -1953 9229 -5188 -5872 -324 -5817 5124 6366 -6453 -4650 -7510 7445 2646 9214 3653 9886 1485 -6653 9243 6124 1092 9037 6700 -7551 5576 2759 2981 2081 6916 -3860 2366 -1423 126 -9828 -1283 2886 -8220 -5501 -4433 4736 4715 6112 6574 -1433 8864 7095 7476 8706 3681 -4232 7195 8056 6397 619 -2537 7413 -9407 5419 395 -4632 4780 -8103 7821 -8515 9965 2842 -2498 -2973 8228 -9093 1600 -6678 5948 -3772 7243 -5785 3169 -3571 1409 6133 961 1311 7607 -7177 9462 -2961 -7214 -3155 3833 8648 -2865 1996 -3808 8614 -8401 -6353 -4598 -3067 5103 1926 2793 5836 -1473 7047 -7147 7696 4585 4686 3957 -8765 160 2344 6459 -1116 -5915 7550 -4101 -8429 -1821 5 4915 -7765 -7230 -1297 -4277 4420 -5836 3273 -5438 -5503 8272 -5914 -8097 4308 -375 -145 7493 -2328 3223 -3525 4059 -2803 3921 6701 -3182 4813 5058 2865 -9529 -3677 -1173 -7884 736 -2940 7108 -9424 -1825 -1763 -184 6826 2291 2283 6795 -9482 3089 -2452 4398 -6521 9197 4116 7515 -4316 -8130 -4671 7107 8642 5191 -5743 -2665 -3273 4172 7301 -8428 2631 -8614 -1856 2146 -5198 9967 -2984 5890 3258 5593 6118 6025 7894 -5230 6646 -6197 4275 -9696 2458 -9620 834 -8965 5233 8000 -4628 3691 9960 -1403 -6886 5941 5501 7598 4521 6750 4605 1317 7847 6406 -507 5137 1480 4702 -5264 -3655 9928 3483 -3381 4238 -4891 -5955 1747 -9498 470 935 1484 -8872 -1736 -5279 -4527 8154 7102 -4051 2808 8285 -7517 4772 -1541 -9639 -9662 -96 -1852 2345 -2359 9017 2961 -6695 -8852 -1236 -4186 -3182 -4771 4670 -140 7972 8323 -3208 -8582 -5998 201 6079 -989 -7586 -4321 -2133 -3503 -7192 -5193 1140 -9799 8647 4482 -8113 -75 4748 302 4339 -3074 -1806 1656 -9090 