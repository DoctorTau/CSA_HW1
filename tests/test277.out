-1788 -4110 -3740 -6430 -2846 -2086 -9204 -2486 -9722 -3106 -4382 -8134 -3996 -3346 -9710 -5448 -3466 -5380 -8944 -7868 -1474 -9942 -5494 -2938 -1206 -766 -3250 -1616 -142 -7834 -2298 -996 -4454 -5224 -3118 -3884 -8218 -4074 -3240 -8942 -192 -4854 -4690 -5040 -134 -1946 -2662 -8268 -324 -7466 -6414 -3920 -3656 -8118 -8154 -7056 -9590 -6550 -3298 -444 -8848 -8224 -9032 -4824 -184 -8356 -6406 -7070 -9614 -8724 -9012 -506 -5106 -7046 -8054 -7114 -806 -6542 -3546 -638 -4694 -788 -2506 -564 -1234 -8178 -7458 -1632 -360 -3138 -5112 -6774 -5188 -1760 -6870 -9872 -5840 -4072 -9314 -6618 -7122 -5188 -5872 -324 -4650 -7510 -3860 -9828 -8220 -4232 -4632 -2498 -6678 -3772 -7214 -3808 -4598 -1116 -7230 -5836 -5438 -5914 -2328 -3182 -7884 -2940 -9424 -184 -9482 -2452 -4316 -8130 -8428 -8614 -1856 -5198 -2984 -5230 -9696 -9620 -4628 -6886 -5264 -9498 -8872 -1736 -9662 -96 -1852 -8852 -1236 -4186 -3182 -140 -3208 -8582 -5998 -7586 -7192 -3074 -1806 -9090 